guides/common/modules/con_installing-smart-proxy-servers.adoc

```diff
@@ -0,0 +1,23 @@
+[id="installing-{smart-proxy-context}-servers_{context}"]
+= Installing {SmartProxyServers}
+
+By installing {SmartProxyServers}, you extend the reach and scalability of your {Project} deployment.
+Setting up a {SmartProxyServer} registers the base operating system on which you are installing to {ProjectServer} and configures the new {SmartProxyServer} to provide the required services within your {Project} deployment.
+
+You can install a {SmartProxyServer} in each of your geographic locations.
+By assigning a {SmartProxy} to each location, you decrease the load on {ProjectServer}, increase redundancy, and reduce bandwidth usage.
+
+[NOTE]
+====
+The maximum number of {SmartProxyServers} that {ProjectServer} can support has no fixed limit.
+It was tested that a {ProjectServer} can support 17 {SmartProxyServers} with 2 vCPUs.
+====
+
+Decide what services you want to enable on each {SmartProxyServer}.
+You can configure the DNS, DHCP, and TFTP services on one of your {SmartProxyServers} or you can use an external server to provide these services to your {SmartProxyServers}.
+
+.Additional resources
+* For information on installing a {SmartProxyServer}, including prerequisites and configuring external services, see {InstallingSmartProxyDocURL}[_{InstallingSmartProxyDocTitle}_].
```

Comment:
```suggestion
* For information on installing {SmartProxyServers}, including prerequisites and configuring external services, see {InstallingSmartProxyDocURL}[_{InstallingSmartProxyDocTitle}_].
```
if you don't mind; else, it'd end up as "a orcharhino Proxy Server" instead of "an ...".

source: https://github.com/theforeman/foreman-documentation/blob/master/guides/common/attributes-base.adoc#L150